### ISSUE TITLE
All zeros for reprs

### ISSUE THREAD

**Issue by doitslow** [SalesforceAIResearch/uni2ts#200]
I am running MoiraiModule.forward on a single variate time series, it turns out that reprs are all zeros.

This looks extremely weird.

Code to reproduce:
`
  MODEL = "moirai"  # model name: choose from {'moirai', 'moirai-moe'}
  SIZE = "small"  # model size: choose from {'small', 'base', 'large'}
  PDT = 6  # prediction length: any positive integer
  CTX = 8  # context length: any positive integer
  PSZ = 4  # patch size: choose from {"auto", 8, 16, 32, 64, 128}
  BSZ = 2  # batch size: any positive integer
  TEST = 20  # test set length: any positive integer
  
  df = pd.read_csv('ts_wide.csv', index_col=0, parse_dates=True)
  ds = PandasDataset(dict(df))
  train, test_template = split(
      ds, offset=-TEST
  )  # assign last TEST time steps as test set
  
  test_data = test_template.generate_instances(
      prediction_length=PDT,  # number of time steps for each prediction
      windows=TEST // PDT,  # number of windows in rolling window evaluation
      distance=PDT,  # number of time steps between each window - distance=PDT for non-overlapping windows
  )
  
  model = MoiraiForecast(
      module=MoiraiModule.from_pretrained("/home/ma-user/work/PRETRAINED_MODELS/moirai-1.1-R-small"),
      prediction_length=PDT,
      context_length=CTX,
      patch_size=PSZ,
      num_samples=100,
      target_dim=1,
      feat_dynamic_real_dim=ds.num_feat_dynamic_real,
      past_feat_dynamic_real_dim=ds.num_past_feat_dynamic_real,
  )
  
  predictor = model.create_predictor(batch_size=BSZ)
  forecasts = predictor.predict(test_data.input)
  
  next(forecasts)
`

**Comment by chenghaoliu89** [comment]
@doitslow patch size should be in the range {8, 16, 32, 64, 128}

**Comment by doitslow** [comment]
Thanks a lot. I overlooked that there is such contraint on patch size. Problem solved.
